app/views/manuals/_manual_section.html.erb

```diff
@@ -9,7 +9,7 @@
       <% if presented_document.body.present? %>
         <div class='body-content-wrapper'>
           <%= render 'govuk_publishing_components/components/govspeak', {} do %>
-            <%= raw(presented_document.body) %>
+            <%= raw(presented_document.body[:intro]) %>
```

Comment:
I was initially worried about passing the entire body plus the broken down body and intro in a single object. On balance though, would this have any noticeable user ramifications? Your reasoning makes sense and would certainly make it easier to manage.

Reply:
What worries you about that? Are you worried it may not be performant enough on the server? We can utilise caching if that helps, e.g. this pseudocode:

```ruby
def intro
  body.strip_out_intro
end

def main
  body.strip_out_main
end

def body
  # the following line is the equivalent to `@body = @body || some_expensive_operation`
  @body ||= some_expensive_operation
end
```

Reply:
I think I assumed there would be a performance hit yeah but I think this was just me making very rough guesses of how ruby worked. Your other point on breaking this apart, plus the pseudo code above mitigates this. I'll sort it out. Cheers!

Reply:
I've broken the methods in this down into `body`, `main` and `intro` where the latter 2 methods consume `body`. It's not the exact implementation you specified above but I think it's a lot neater than it was. Let me know if this is a problem and I can re-amend.